ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake

```diff
@@ -40,7 +40,7 @@ function(ament_target_dependencies target)
     message(FATAL_ERROR "ament_target_dependencies() the first argument must be a valid target name")
   endif()
   if(${ARGC} GREATER 0)
-    cmake_parse_arguments(ARG "INTERFACE;PUBLIC" "" "" ${ARGN})
+    cmake_parse_arguments(ARG "INTERFACE;PUBLIC;SYSTEM" "" "" ${ARGN})
```

Comment:
@andre-nguyen consider updating the documentation block right above.

Reply:
done